recipes/irida-uploader/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "irida-uploader" %}
+{% set version = "0.1" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/phac-nml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 8080b411c0fcbc40d7792c65f1c9736186b0f5f436dc7115e0979cb80de1ea3e
+  patches: 
+    - upload_run.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python =3.6
+    - pip
+    - make
```

Comment:
`make` is the only dependency that should be in the `build:` section. This also suggests that you need some compilers (e.g., `{{ compiler("c") }}`)

Reply:
I've removed make for now. Future versions of the recipe may require it, though.